pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -225,6 +225,10 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 				return nil, fmt.Errorf("running with swap on is not supported, please disable swap! or set --fail-swap-on flag to false. /proc/swaps contained: %v", swapLines)
 			}
 		}
+	} else {
+		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.NodeSwap) && !cgroups.IsCgroup2UnifiedMode() {
+			return nil, fmt.Errorf("running swap with cgroups v1 is not supported. please either disable %s feature gate", kubefeatures.NodeSwap)
```

Comment:
```suggestion
			return nil, fmt.Errorf("running swap with cgroups v1 is not supported. Please either disable %s feature gate or start using cgroup v2.", kubefeatures.NodeSwap)
```

Reply:
Thanks :) Probably my mind had a context switch and didn't switch back..

Reply:
This is not addressed yet but outdated for rebasing.